packages/autometrics/src/exporter-prometheus-push-gateway/fetch.ts

```diff
@@ -0,0 +1,3 @@
+const fetch = global.fetch;
+
+export { fetch };
```

Comment:
Added this to a separate file so that we can easily replace the file with `node-fetch-native` when building the Node bundles.